src/main/scala/tech/cryptonomic/conseil/db/DatabaseQueryExecution.scala

```diff
@@ -2,13 +2,13 @@ package tech.cryptonomic.conseil.db
 
 import tech.cryptonomic.conseil.tezos.{ApiFiltering, Tables, TezosDatabaseOperations}
 import tech.cryptonomic.conseil.tezos.ApiOperations._
-
 import slick.jdbc.PostgresProfile.api._
+
 import scala.language.higherKinds
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 /** database-specific filter operations support */
-object DatabaseQueryExecution {
+object DatabaseQueryExecutionTypes {
```

Comment:
Is this useful or needed in some way?

Reply:
avoid naming conflicts  - so here I had to rename it as I've extracted the class, there was also trait named `DatabaseQueryExecution`

Reply:
I can't find where the class with conflicting name is?

Reply:
it's conflicting with the trait

Reply:
I see no class `DatabaseQueryExecution`, only the trait and the object, I must be missing something...?

Reply:
yes, it was conflicting with the trait, but it is not relevant any more